src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -9855,12 +9863,12 @@ namespace AirflowNetworkBalanceManager {
             }
             NodeFound.deallocate();
 
-            // Asign AirLoop Number to every node and linkage
+            // Assign AirLoop Number to every node and linkage
             // Zone first
             for (i = 1; i <= AirflowNetworkNumOfZones; i++) {
                 for (j = 1; j <= NumOfZones; j++) {
                     if (!ZoneEquipConfig(j).IsControlled) continue;
-                    if (MultizoneZoneData(i).ZoneNum == j) {
+                    if ((MultizoneZoneData(i).ZoneNum == j) && (ZoneEquipConfig(j).NumInletNodes > 0)) {
```

Comment:
And so we don't need any ELSE blocks to accommodate this change in logic?  We don't need (If zoneNum is J but numInletNodes is zero)?